nodes/Bento/Bento.node.ts

```
@@ -146,7 +222,326 @@ export class Bento implements INodeType {
+					},
+					{
+						name: 'Unsubscribe',
+						value: 'unsubscribe',
+						description: 'Unsubscribe the email address',
+					},
+					{
+						name: 'Change Email',
+						value: 'change_email',
+						description: 'Change the email address of the subscriber',
+					},
+				],
```

ESLint

[error] Alphabetize by 'name'. Order: Add Field | Add Tag | Add Tag via Event | Change Email | Remove Field | Remove Tag | Subscribe | Unsubscribe [autofixable]

(n8n-nodes-base/node-param-options-type-unsorted-items)